lib/convection/model/template/resource/aws_elbv2_listener_rule.rb

```diff
@@ -0,0 +1,35 @@
+require_relative '../resource'
+
+module Convection
+  module Model
+    class Template
+      class Resource
+        ##
+        # AWS::ElasticLoadBalancingV2::ListenerRule
+        ##
+        class ELBV2ListenerRule < Resource
+          type 'AWS::ElasticLoadBalancingV2::ListenerRule', :elbv2_listener_rule
+          # http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-elasticloadbalancingv2-listenerrule.html
+          property :actions, 'Actions', :type => :list
+          property :conditions, 'Conditions', :type => :list
+          property :listener_arn, 'ListenerArn'
+          property :priority, 'Priority'
+
+          # Append an action
+          def action(&block)
+            action = ResourceProperty::ELBV2ListenerRuleAction.new(self)
+            action.instance_exec(&block) if block
+            actions << action
+          end
+
+          # Append a condition
+          def rule_condition(&block)
+            condition = ResourceProperty::ELBV2ListenerRuleCondition.new(self)
+            condition.instance_exec(&block) if block
+            conditions << condition
+          end
```

Comment:
@bturner-r7 @rdickey *beware of dragons* eek, @bcall-r7 and I discovered that having overriding `condition` will cause issues rendering the CF JSON (there is a top level `condition` method that is override if you re-define a property of that name). That condition method is used to fill the `Conditions` CF field.

@bcall-r7 did I summarise that right? I can't recall exactly what scenario you exposed that issue under.